server/boot/extend-models.js

```diff
@@ -18,4 +20,27 @@ module.exports = function(server) {
   serviceMetric(server.models.ServiceMetric);
   serviceProcess(server.models.ServiceProcess);
   gateway(server.models.Gateway);
+  var modelsToWatch = [
+    {name: 'agenttrace', instance: agentTrace},
+    {name: 'executor', instance: executor},
+    {name: 'expressusagerecord', instance: expressUsageRecord},
+    {name: 'profiledata', instance: profileData},
+    {name: 'serviceinstance', instance: serviceInstance},
+    {name: 'serverservice', instance: serverService},
+    {name: 'servicenetric', instance: serviceMetric},
+    {name: 'serviceprocess', instance: serviceProcess},
+    {name: 'gateway', instance: gateway},
+  ];
+  server.on('ready', function(dbWatcher) {
```

Comment:
This should probably also check if dbWatcher is passed in. It would be null for non-postgres cases...right?